source/NVDAObjects/window/winword.py

```diff
@@ -1430,10 +1466,520 @@ def _get_WinwordWindowObject(self):
+			# Translators: A column in a column chart.
+			text=pgettext('chart','column')
+		elif chartType in (_chartConstants.xl3DLine, _chartConstants.xlLine, _chartConstants.xlLineMarkers, _chartConstants.xlLineMarkersStacked, _chartConstants.xlLineMarkersStacked100, _chartConstants.xlLineStacked, _chartConstants.xlLineStacked100):
+			# Translators: A data point in a line chart.
+			text=_("data point")
+		else:
+			# Translators: A segment of a chart for charts which don't have a specific name for segments.
+			text=_("item")
+		return text
+
+	def invokeChartElement(self, keyIndex):
+		val=self.chartElements[self.keyList[keyIndex]]
+		func=val[0]
+		arg=val[1]
+		if arg==None:
+			func()
+		else:
+			func(arg)
+		
+	def script_nextChartElement(self,gesture):
+		if self.keyIndex == self.elementsCount-1:
+			self.keyIndex=0
+		else:
+			self.keyIndex=self.keyIndex+1
+		self.invokeChartElement(self.keyIndex)
+	script_nextChartElement.canPropagate=True
+
+	def script_previousChartElement(self,gesture):
+		if self.keyIndex == 0:
+			self.keyIndex=self.elementsCount-1
+		else:
+			self.keyIndex=self.keyIndex-1
+		self.invokeChartElement(self.keyIndex)
+	script_previousChartElement.canPropagate=True
+
+	def script_activatePosition(self,gesture):
+		# Toggle browse mode pass-through.
+		self.passThrough = True
+		self.ignoreTreeInterceptorPassThrough=False
+		browseMode.reportPassThrough(self)
+	# Translators: Input help mode message for toggle focus and browse mode command in web browsing and other situations.
+	script_activatePosition.__doc__=_("Toggles between browse mode and focus mode. When in focus mode, keys will pass straight through to the application, allowing you to interact directly with a control. When in browse mode, you can navigate the document with the cursor, quick navigation keys, etc.")
+	script_activatePosition.category=inputCore.SCRCAT_BROWSEMODE
+
+	def script_disablePassThrough(self, gesture):
+		rangeStart=self.wordApplicationObject.ActiveDocument.Range().Start
+		self.wordApplicationObject.ActiveDocument.Range(rangeStart, rangeStart).Select()
+		eventHandler.executeEvent("gainFocus", api.getDesktopObject().objectWithFocus())
+
+	__gestures = {
+				"kb(laptop):upArrow":"nextChartElement",
+				"kb(desktop):upArrow":"nextChartElement",
+				"kb(laptop):downArrow":"previousChartElement",
+				"kb(desktop):downArrow":"previousChartElement",
+				"kb(desktop):leftArrow":"previousChartElement",
+				"kb(laptop):leftArrow":"previousChartElement",
+				"kb(desktop):rightarrow":"nextChartElement",
+				"kb(laptop):rightArrow":"nextChartElement",
+				"kb:enter": "activatePosition",
+				"kb(desktop):numpadEnter":"activatePosition",
+				"kb:space": "activatePosition",
+				"kb:escape": "disablePassThrough",
+	}
+
+class WordChartSeriesTrendline(WordChart):
+	_trendlineTypeMap = {_chartConstants.xlExponential: 'Exponential',

```

Comment:
Added translation comments, as suggested.
Changed code is in branch in_t1541_Fin
Thanks a lot for the code review.